lib/Backend/Lower.cpp

```diff
@@ -14977,7 +15012,7 @@ Lowerer::GenerateFastStringLdElem(IR::Instr * ldElem, IR::LabelInstr * labelHelp
 }
 
 bool
-Lowerer::GenerateFastLdElemI(IR::Instr *& ldElem, bool *instrIsInHelperBlockRef)
+Lowerer::GenerateFastLdElemI(IR::Instr *& ldElem, bool *instrIsInHelperBlockRef, IR::LabelInstr ** bailOutLabelInstr)
```

Comment:
Removed. Done

---
In reply to: [67765060](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67765060) [](ancestors = 67765060)